src/components/PersonTabs/PeopleTabs.tsx

```diff
@@ -0,0 +1,121 @@
+import { Loader } from '../Loader';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
```

Comment:
The import path '././types' seems incorrect. It should be './types' to correctly resolve the module path.